src/components/icon-as-button/IconAsButton.jsx

```diff
@@ -1,9 +1,8 @@
-// @flow
+// @flow strict
 import React from 'react';
```

Comment:
@PinkaminaDianePie I am not sure, but I think we have to ways of importing react `import * as React from 'react';`. Maybe we should unify it?

Reply:
Yeah, i'd like to unify it as `import * as React from 'react';`. Maybe in next PR :)